src/through.ml

```diff
@@ -0,0 +1,36 @@
+module Make (C : S.Cache) (DB : S.DB with type key = C.key) = struct
+  type 'a t = 'a C.t
+  type 'a d = 'a DB.d
+  type key = C.key
+
+  let v c = C.v c
+  let stats t = C.stats t
+  let is_empty t = C.is_empty t
+  let capacity t = C.capacity t
+  let size t = C.size t
+  let clear t = C.clear t
+  let mem t d k = C.mem t k || DB.mem d k
+  let find t d k = try C.find t k with Not_found -> DB.find d k
```

Comment:
If `k` is in db but not in cache, shouldn't `k` be inserted in cache? (same for `mem`)

Reply:
yes it is a good, idea!